app/src/main/java/com/dicoding/picodiploma/sejiwaproject/features/match/next/NextMatchFragment.kt

```diff
@@ -0,0 +1,101 @@
+package com.dicoding.picodiploma.sejiwaproject.features.match.next
+
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.dicoding.picodiploma.sejiwaproject.R
+import com.dicoding.picodiploma.sejiwaproject.commons.api.ApiRepository
+import com.dicoding.picodiploma.sejiwaproject.commons.utils.invisible
+import com.dicoding.picodiploma.sejiwaproject.commons.utils.visible
+import com.dicoding.picodiploma.sejiwaproject.features.league.detail.page.PageViewModel
+import com.dicoding.picodiploma.sejiwaproject.features.match.next.model.NextMatch
+import com.google.gson.Gson
+import kotlinx.android.synthetic.main.fragment_next_match.*
+
+/**
+ * A simple [Fragment] subclass.
+ */
+class NextMatchFragment : Fragment(),
+    NextMatchView {
+    private lateinit var rvMatch: RecyclerView
+    private lateinit var pageViewModel: PageViewModel
+    private lateinit var presenter: NextMatchPresenter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        pageViewModel = ViewModelProviders.of(this).get(PageViewModel::class.java).apply {
+            setIndex(arguments?.getInt(ARG_SECTION_NUMBER) ?: 2)
+        }
+    }
+
+    override fun onCreateView(
```

Comment:
The onCreateView function should only be used to inflate view. For initializing the view, it should be done inside the onViewCreated function.

Reply:
resolveld, thanks